src/main/scala/com/amazon/deequ/VerificationResult.scala

```diff
@@ -144,7 +145,7 @@ object VerificationResult {
     val constraint = constraintResult.constraint
     constraint match {
       case asserted: RowLevelAssertedConstraint =>
-        constraintResult.metric.flatMap(metricToColumn).map(asserted.assertion(_))
+        constraintResult.metric.flatMap(metricToColumn).map(asserted.assertion(_)).orElse(Some(lit(false)))
```

Comment:
I probably need to look at the code again, but could you remind me the difference between `RowLevelAssertedConstraint` and `RowLevelConstraint` / why we don't need to add it for the latter? Are `RowLevelConstraint` for checks like Completeness, Uniqueness where row level results are boolean and `RowLevelAssertedConstraint` are for checks like Minimum, Sum where we need to compare the row-level value to a given assertion?